Blog: Altinity announcement (#3107)
PostHog/posthog.com: contents/blog/posthog-altinity-announce.md

> ---

Reviewer: I think we should also clearly link to our documentation on actually deploying using altinity cloud here - or do you think the link in marketplace is sufficient?

Reviewer: That's the correct one :)

> date: 2022-03-07

Reviewer 2: ```suggestion
date: 2022-03-09
```

title: PostHog teams up with Altinity
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author: ["joe-martin"]
featuredImage: ../images/blog/posthog-engineering-blog.png
featuredImageType: full

> categories: ["Release notes", "Product updates"]

Reviewer 2: ```suggestion
categories: ["Product updates"]
```

---

That’s right, we’re teaming up with Altinity. It’s an exciting new partnership that also marks the launch of the [PostHog Marketplace](/marketplace). Double announcement!

Going forward, the marketplace is where you’ll be able to find third-party services which layer on top of PostHog, such as companies offering support, integration or development services. Altinity is the first such company, but there are more to come in the future. 

You may know [Altinity](https://altinity.com/) as experts in all things ClickHouse and data infrastructure. That’s why we’re working with them! Going forward, Altinity will be offering a variety of services to help give large organizations on PostHog an increased level of support for self-hosted deployments.

This increased support includes:

- 24/7 technical support for ClickHouse in any environment
- Bespoke training for analytics developers and system administrators
- Altinity Stable builds for ClickHouse

> In addition to providing training and support, Altinity can also offer [Altinity.Cloud](https://altinity.com/cloud-database/) as a backend for PostHog users who prefer a private cloud environment. This essentially means Altinity can manage your ClickHouse deployment for you, turning PostHog into a fully managed service.

Author: ```suggestion
In addition to providing training and support, Altinity can also offer [Altinity.Cloud as a backend](/docs/self-host/configure/using-altinity-cloud) for PostHog users who prefer a private cloud environment. This essentially means Altinity can manage your ClickHouse deployment for you, turning PostHog into a fully managed service.
```
